draft-wang-ppm-dap-taskprov.md

```diff
@@ -422,8 +439,11 @@ with "invalidTask".
 > OPEN ISSUE: In case of opt-out, would it be useful to specify how to report
 > this to the Author?
 
-Finally, the Helper completes the aggregate initialize request as usual, deriving the VDAF
-verification key for the task as described in {{vdaf-verify-key}}.
+Finally, the Helper completes the aggregate initialize request as usual,
+deriving the VDAF verification key for the task as described in
+{{vdaf-verify-key}}. During the process, if Helper's report share does not
+present a `task_prov` extension type, Helper SHOULD abort the aggregate request
+with "invalidMessage".
```

Comment:
What's the intent here? To ensure that all Clients indicate support for `task_prov`? I think this should be a MUST.

Reply:
The intent is to keep consistency, if the client opt in to taskprov, it should set both the ext type and dap-taskprov header. agree with the must